client/src/Components/Layout/Login/Login.jsx

```diff
@@ -0,0 +1,331 @@
+import React, { Component } from 'react';
+import queryString from 'query-string';
+import { Link, Redirect } from 'react-router-dom';
+import {
+  Grid,
+  Box,
+  Typography,
+  Button,
+  Paper,
+  TextField,
+  FormHelperText,
+  FormControl,
+  Input,
+  InputLabel,
+  IconButton,
+  InputAdornment,
+} from '@material-ui/core';
+import {
+  Person as PersonIcon,
+  Lock as LockIcon,
+  Visibility,
+  VisibilityOff,
+} from '@material-ui/icons';
+import Cookies from 'js-cookie';
+import { withStyles } from '@material-ui/core/styles';
+
+import LoaderProgress from '../../Common/LoaderProgress';
+
+import axios from 'axios';
+import LoginStyles from './LoginStyle';
+
+class Login extends Component {
+  state = {
+    logindata: { email: null, password: null },
+    isLoading: false,
+    displayBlock: true,
+    showPassword: false,
+    email: {
+      Message: '',
+      isValid: true,
+    },
+    password: {
+      Message: '',
+      isValid: true,
+      minLength: 3,
+    },
+    redirect: false,
+    ReturnUrlText: '/',
+    SignupLink: '/user/Signup/',
```

Comment:
Will this ever need to be changed while the app is running? 

If not, I would move it out of the component state. Maybe create a constants file with these URLs in, that you can import into your components, so that if you update the paths later you only need to change them in one place